README.md

```diff
@@ -79,7 +79,62 @@ Yelp:    0.69 (+2%)
 ```
 
 ---
+### How it works
+#### AFINN 
+AFINN is a list of words rated for valence with an integer between minus five (negative) and plus five (positive). Sentiment analysis is performed by cross-checking the string tokens( words, emojis) with the AFINN list and getting their respective scores. The comparative score is simply: sum of each token / number of tokens. So for example let's take the following:
```

Comment:
Couple minor nitpicks here:
- Fix spacing / formatting in parenthetical "tokens (e.g. words and emojis)"
- Format "sum of each token / number of tokens" as code by wrapping in `code`